test/integration/action/action-integration.spec.js

```diff
@@ -176,6 +180,7 @@ describe('Action Integration Tests', function() {
     channels2 = new ChannelAction(store2, grpc2, nav2, notify2);
     invoice2 = new InvoiceAction(store2, grpc2, nav2, notify2);
     payments2 = new PaymentAction(store2, grpc2, nav2, notify2);
+    autopilot2 = new AtplAction(store2, grpc2, db2, notify2);
```

Comment:
I know uniformity is good, but wanted to point out this isn't used anywhere. I think that's alright though.